examples/multiwindow.rs

```diff
@@ -1,63 +1,73 @@
 use wry::Result;
-use wry::{Application, Attributes, Callback};
+use wry::{Application, Attributes, WindowProxy, RpcRequest};
+use serde_json::Value;
 
 fn main() -> Result<()> {
     let mut app = Application::new()?;
 
+    let html = r#"
+<script>    
+async function openWindow() {
+    await window.rpc.notify("openWindow", "https://i.imgur.com/x6tXcr9.gif");
+}
+</script>
+<p>Multiwindow example</p>
+<button onclick="openWindow();">Launch window</button>        
+"#;
+
     let attributes = Attributes {
-        url: Some("https://tauri.studio".to_string()),
+        url: Some(format!("data:text/html,{}", urlencoding::encode(html))),
```

Comment:
```suggestion
        url: Some(format!("data:text/html,{}", html)),
```
It won't be rendered on windows if it's encoded. Looks like the web engine will do this for us.
Could you remove this dependency?

Reply:
Sure thing, that's done! Strange they accept it un-encoded, I assume the browser is decoding before passing to the webview, all good :+1: